README.md

```diff
@@ -16,7 +16,7 @@ ts-jest@24.0.0
 ## Installation
 
 - A Transformer needs to be provided at compile time.
-We need to tell ts-jest to use ttypescript that allow us to use a transformer.
+We need to tell ts-jest to use typescript that allow us to use a transformer.
```

Comment:
It's actually ttypescript. Ts-jest needs to use ttypescript compiler in order to enable transformers.  https://github.com/cevek/ttypescript. Shall we add the link next to it so it will be easier to understand?